tests/testthat/test-styler.R

```diff
@@ -20,3 +20,11 @@ test_that("removes space at EOL", {
 test_that("removes blank lines at EOF", {
   expect_equal(style_text(c("a() ", "", "")), "a()")
 })
+
+test_that("logical T and F are converted to TRUE and FALSE", {
+  expect_equal(style_text("T == F", c("TRUE = FALSE")))
```

Comment:
You are right, that actually might be rather tricky. Since this goes slightly beyond the scope of a simple test, I opened #3